partials/featured-posts.hbs

```diff
@@ -0,0 +1,32 @@
+{{#get "posts" filter="featured:true" limit="all"}}
+    {{#if posts}}
+        <section class="featured-wrapper container">
+            <div class="title-wrap">
+                {{#if @custom.featured_title}}
+                    <h4>{{t "Featured Articles"}}</h4>
+                {{/if}}
+                <a href="/tag/featured" class="view-all">
```

Comment:
After importing routes.yaml, this URL gives a 404. Is `featured:true` really a tag, or do we need a dedicated template (like you did with blog.hbs?)